stubs/inertia-react-ts/resources/js/ssr.tsx

```diff
@@ -4,7 +4,7 @@ import createServer from '@inertiajs/react/server';
 import { resolvePageComponent } from 'laravel-vite-plugin/inertia-helpers';
 import route from '../../vendor/tightenco/ziggy/dist/index.m';
 
-const appName = 'Laravel';
+const appName = import.meta.env.VITE_APP_NAME || 'Laravel';
```

Comment:
ssr can also use `process.env` to access envs (even without the `VITE_` prefix)

Reply:
I like that it is consistent and uses `VITE_APP_NAME`, otherwise if you remove that ENV it could work in SSR and not a normal build.

Reply:
I agree. Using `process.env` would be a good example of accessing secrets/api keys and not exposing them to the frontend, but using it here would be a little misleading